src/serlio/modifiers/MayaCallbacks.cpp

```diff
@@ -414,13 +414,18 @@ std::filesystem::path getAssetDir() {
 }
 
 void detectAndAppendCGACErrors(prt::CGAErrorLevel level, const wchar_t* message, CGACErrors& cgacErrors) {
-	if (message != nullptr && (std::wcsstr(message, L"CGAC version") || std::wcsstr(message, L"Non-recognized builtin method"))) {
-		const bool shouldBeLogged =
-		        (std::wcsstr(message, L"newer than current") || (level == prt::CGAErrorLevel::CGAERROR));
-
+	if (message != nullptr) {
+		bool shouldBeLogged = (level == prt::CGAErrorLevel::CGAERROR);
 		std::wstring stringMessage = message;
-		prtu::replaceCGACWithCEVersion(stringMessage);
-		cgacErrors.emplace_back(level, shouldBeLogged, stringMessage);
+
+		if (std::wcsstr(message, L"CGAC version")) {

```

Comment:
Consider explicitly writing the ` != nullptr` check for `std::wcsstr`

Please update all wcsstr occurrences.